index.html

```diff
@@ -2,20 +2,134 @@
+                        <p class="read-more">
+                            <a href="https://www.ecpg.eu/">Visit Site</a>
+                        </p>
+                    </div>
+                </div>
+                <div class="work-card alt">
+                    <div class="meta">
+                        <img src="assets/img/workcard-backgrounds/9.jpg" class="photo" alt="photo of colored stucco-like texture">
+                    </div>
+                    <div class="description"> 
+                        <h2>Site Redesign</h2>
+                        <h1>Bath Institute for Rheumatic Disease</h1>
+                        <p>Bespoke site redesign and responsive development using WordPress.</p>
+                        <p class="read-more">
+                            <a href="http://www.birdbath.org.uk/">Visit Site</a>
+                        </p>
+                    </div>
+                </div>
+                <div class="work-card">
+                    <div class="meta">
+                        <img src="assets/img/workcard-backgrounds/33.jpg" class="photo" alt="photo of colored stucco-like texture">
+                    </div>
+                    <div class="description"> 
+                        <h2>Landing Page</h2>
+                        <h1>Brass Mill Media</h1>
+                        <p>Concept design and static site development.</p>
+                        <p class="read-more">
+                            <a href="https://www.brassmillmedia.com">Visit Site</a>
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="about">
+            <div class="details">
+                <h1><span class="block">Ab</span>out<span class="dot">.</span></h1>
+                <p>I'm a fullstack developer and reluctant designer who is originally from Texas but now lives on a 60-foot narrowboat in Southwest England.</p>
+                <p>I'm a highly adaptable and resilient problem solver with a strong background in developing others.</p>
+                <p>I am ambitious, extremely organised and a team player. With one years freelance web development experience, I am looking to join a team as a junior developer in a role that could potentially evolve to training or mentoring others.</p>
+            </div>
+            <div class="img">
+                <img class="texas block" src="assets/img/Texas.svg" alt="black outline of state of Texas">
+            </div>
+        </section>
+
+        <section class="contact">
+            <div class="title">
+                <h2>Want to work with me?</h2>
+                <h1>Get in <span class="block">touch<span class="dot">.</span></span></h1>
+            </div>
+            <div class="contact-details">
+                <address>
+                    <p>audreyhayes<span class="highlight">(zero-as-a-digit)(at)</span>gmail<span class="highlight">(dot)</span>com</p>
```

Comment:
could also add a` mailto: ` link for easier email contact

Reply:
terrified of spam bots, so probably won't have my real address anywhere in the DOM where it can be found 😄 This is probably overkill, but better make someone do some typing then get a crap ton of emails I don't want haha